docs/content/Caching/Getting-Started-Pre-Aggregations.md

```diff
@@ -159,7 +159,175 @@ the pre-aggregations are rebuilt.
 These refreshes can be done on-demand, or [in the background as a scheduled
 process][ref-caching-preaggs-bk-refresh].
 
+## Ensuring pre-aggregations are targeted by queries
+
+### Additivity
+
+So far, we've described pre-aggregations as aggregated versions of your existing
+data.
+
+However, there are some rules that apply to when Cube.js uses the
+pre-aggregation. The "additivity" of fields specified in both the query and in
+the pre-aggregation determine this.
+
+So what is additivity? Let's add another cube called `LineItems` to the previous
+example to demonstrate. The `LineItems` "belong to" the `Orders` cube, and are
+[joined][ref-schema-joins] as such:
+
+```javascript
+cube(`LineItems`, {
+  sql: `SELECT * FROM public.line_items`,
+
+  joins: {
+    Orders: {
+      sql: `${CUBE}.order_id = ${Orders}.id`,
+      relationship: `belongsTo`,
+    },
+  },
+
+  measures: {
+    count: {
+      type: `count`,
+      drillMembers: [id, createdAt],
+    },
+  },
+
+  dimensions: {
+    id: {
+      sql: `id`,
+      type: `number`,
+      primaryKey: true,
+    },
+
+    createdAt: {
+      sql: `created_at`,
+      type: `time`,
+    },
+  },
+});
+```
+
+Some sample data from the `line_items` table might look like:
+
+| **id** | **product_id** | **order_id** | **quantity** | **price** | **profit_margin** | **created_at**             |
+| ------ | -------------- | ------------ | ------------ | --------- | ----------------- | -------------------------- |
+| 1      | 31             | 1            | 1            | 275       | 1                 | 2019-01-31 00:00:00.000000 |
+| 2      | 49             | 2            | 6            | 248       | 0.1               | 2021-01-20 00:00:00.000000 |
+| 3      | 89             | 3            | 6            | 197       | 0.2               | 2021-11-25 00:00:00.000000 |
+| 4      | 71             | 4            | 8            | 223       | 0.15              | 2019-12-23 00:00:00.000000 |
+| 5      | 64             | 5            | 5            | 75        | 0.7               | 2019-04-20 00:00:00.000000 |
+
+Looking at the raw data, we can see that if the data were to be aggregated by
+day (for example), then we could simply add together the `quantity` and `price`
+fields. This means that `quantity` and `price` are both "additive measures", and
+we can represent them in the `LineItems` schema as follows:
+
+```javascript
+cube(`LineItems`, {
+  ...,
+  measures: {
+    ...,
+    quantity: {
+      sql: `quantity`,
+      type: `sum`,
+    },
+    price: {
+      type: `sum`,
+      sql: `price`,
+      format: `currency`,
+    },
+  },
+  ...,
+});
+```
+
+Because neither `quantity` and `price` reference any other measures in our
+`LineItems` cube, we can also say that they are "additive leaf measures". Any
+query requesting only these two measures can be called a "leaf measure additive"
+query.
+
+### Non-Additivity
```

Comment:
Maybe it's good to have a one-liner here: "If your measure has none one of these types ... but likely has these ones ..., it's non-additive. Here's what it means."

Reply:
Same as above